nextflow.config

```
@@ -56,6 +56,11 @@ params {
 
     arg_skip_rgi                        = false
     arg_rgi_savetmpfiles                = false
+    arg_rgi_alignmenttool               = 'BLAST'
+    arg_rgi_includeloose                = false
```

The RGI docs say about the `loose` algorithm, that "Combined with phenotypic screening, the Loose algorithm allows researchers to hone in on new AMR genes.". Shouldn't we include it by default to get the highest number of possible results?
```suggestion
    arg_rgi_includeloose                = true
```